services/sources/publisher_backlist_service.py

```diff
@@ -0,0 +1,115 @@
+from datetime import datetime, timedelta, timezone
+import os
+import requests
+import json
+import urllib.parse
+from typing import Optional, Dict
+
+from logger import create_log
+from mappings.UofM import UofMMapping
+from .source_service import SourceService
+
+
+logger = create_log(__name__)
+
+BASE_URL = "https://api.airtable.com/v0/appBoLf4lMofecGPU/Publisher%20Backlists%20%26%20Collections%20%F0%9F%93%96?view=UofMichigan%20Press"
```

Comment:
Based on our conversation on Wednesday - do we want to update this to look at the all lists view?

Reply:
I'm fine with changing the view to All lists view since we clarified which view we will be using for the process in the future.